_config.yml

```diff
@@ -14,18 +14,11 @@ execute:
   allow_errors              : false # If `False`, when a code cell raises an error the execution is stopped, otherwise all cells are always run.
   stderr_output             : show  # One of 'show', 'remove', 'remove-warn', 'warn', 'error', 'severe'
 
-# Information about where the book exists on the web
-repository:
-  url: https://bayesiancomputationbook.com  # Online location of your book
```

Comment:
Im not expert in jupyter book but I think we need this right?

Reply:
Not sure was the correct way is either, but  the book renders properly both locally and at the github preview. Also before the github icon was incorrectly pointing the to welcome page of the book instead of to the github repo.